extension/content_script.js

```diff
@@ -273,6 +275,8 @@ function scrollToTracker(cb) {
 	// and that can be jarring. Be a little forgiving, let them "walk into" the reading band, like
 	// first level in Mario: https://www.youtube.com/watch?v=K-NBcP0YUQI. Experiment w value
 	const topPageForgiveness = 50;
+	// Above this is the reading timer; when scrolling up, don't let tracker get above this
+	const upperLimit = 35;
```

Comment:
This is actually a separate issue; if only scroll up when user is going backward at top of page, we will have some text that is highlighted obscured by the reading timer.

Reply:
Nice catch!